director/scripts/test/JSON_Visualization_README.txt

```diff
@@ -0,0 +1,40 @@
+This documentation explains the different parts of the JSON file needed for the
+visualization of data during a simulation:
+
+"model_file": This is the directory containing the file for the robot/plant
+              description (like an urdf file).
+
+"weld-body": This is whether the user wants "WeldFrames()" to be called on the
+             robot/plant. If this property does not exist then the "WeldFrames()"
+             function will not be called
+
+"data": This will be a list of all the different visualizations/massages that
+        the user wants to be displayed. The 3 types of visualizations will be
+        kinematic data, the center of mass, and an LCM message. So far there
+        are 2 types of visualizations that can be shown both of which are
+        kinematic data:
+
+        1) A trace line which has the following properties:
+
+        "name": The line's name to be displayed in the GUI
+        "body_part": The name of the body part/context on which the line would be
```

Comment:
suggest "frame" instead of "body_part" for this field, since that's what we actually use.